qDict/src/main/res/layout/about.xml

```diff
@@ -4,56 +4,73 @@
                 android:layout_width="match_parent"
                 android:layout_height="match_parent"
                 android:background="@color/mmt_white"
-                android:paddingBottom="15dp">
+                android:paddingBottom="15dp"
+                xmlns:app="http://schemas.android.com/apk/res-auto">
+
+    <androidx.constraintlayout.widget.ConstraintLayout
+        android:id="@+id/icon_name_constraint_layout"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content">
 
     <ImageView
         android:id="@+id/iv_icon"
-        android:layout_width="wrap_content"
+        android:layout_width="0dp"
         android:layout_height="wrap_content"
-        android:layout_marginLeft="@dimen/about_padding"
-        android:layout_marginTop="@dimen/large_padding"
         android:contentDescription="@string/app_name"
-        android:src="@drawable/ic_launcher">
+        android:src="@drawable/ic_launcher"
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintEnd_toStartOf="@+id/tv_app_name"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent"
+        app:layout_constraintWidth_percent="0.3">
     </ImageView>
 
     <com.mmt.widget.QTextView
         android:id="@+id/tv_app_name"
-        android:layout_width="wrap_content"
+        android:layout_width="0dp"
         android:layout_height="wrap_content"
-        android:layout_alignTop="@id/iv_icon"
-        android:layout_marginLeft="18dp"
-        android:layout_marginTop="2dp"
-        android:layout_toRightOf="@id/iv_icon"
+        android:gravity="center"
         android:text="@string/app_name"
         android:textAppearance="?android:attr/textAppearanceLarge"
-        android:textColor="@color/mmt_black"/>
+        android:textColor="@color/mmt_black"
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toEndOf="@+id/iv_icon"
+        app:layout_constraintTop_toTopOf="parent"
+        app:layout_constraintWidth_percent="0.3" />
 
+    <LinearLayout
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        app:layout_constraintHorizontal_bias="0.5"
+        app:layout_constraintStart_toEndOf="@+id/iv_icon"
+        app:layout_constraintTop_toBottomOf="@+id/tv_app_name"
+        app:layout_constraintEnd_toEndOf="parent"
+        >
```

Comment:
formatting

Reply:
OK, I will deal with this problem later. :)

Reply:
Simply update the PR when you have your changes ready or comment here if you have any doubts.